src/Raven.Studio/typescript/models/resources/serverDashboard/driveUsage.ts

```diff
@@ -60,10 +60,8 @@ class driveUsage {
     ravendbToUsedSpacePercentage: KnockoutComputed<number>;
 
     gridController = ko.observable<virtualGridController<driveUsageDetails>>();
-    private colorClassProvider: (name: string) => string;
     
-    constructor(dto: Raven.Server.Dashboard.MountPointUsage, colorClassProvider: (name: string) => string, includeTemp: KnockoutObservable<boolean>) {
-        this.colorClassProvider = colorClassProvider;
+    constructor(dto: Raven.Server.Dashboard.MountPointUsage, includeTemp: KnockoutObservable<boolean>, private storagePieChart: storagePieChart ) {
```

Comment:
why do we need to provide entire storePieChart? Can we provide colorClassProvider only ?

Reply:
done